GraphicsView/demo/Alpha_shapes_2/Alpha_shapes_2.cpp

```diff
@@ -6,7 +6,7 @@
 #include <CGAL/Alpha_shape_2.h>
 #include <CGAL/Alpha_shape_face_base_2.h>
 #include <CGAL/Alpha_shape_vertex_base_2.h>
-#if BOOST_VERSION >= 105600
+#if BOOST_VERSION >= 105600 || BOOST_GCC < 40500
```

Comment:
Your commit is entitled "Also disable WKT if gcc version is <4.5 " but the logic of the preprocessor code contradicts the title.

And what will happen when `BOOST_GCC` is not defined? In expressions, the preprocessor evaluates undefined macros to 0, and '0 < 40500' is always true.

If the line was:
```c++
#if BOOST_VERSION >= 105600 || BOOST_GCC >= 40500
```
then the meaning would be:
  - if Boost >=1.56
  - or the compiler is gcc>=4.5 (because for non-gcc compilers, the expression would be `0 >= 40500`).

The right test is:
```c++
#if BOOST_VERSION >= 105600 && (! defined(BOOST_GCC) || BOOST_GCC >= 40500)
```

Reply:
oO I must have been tired yesterday !